app/workers/webhooks.php

```diff
@@ -68,7 +70,7 @@ protected function execute(array $events, string $payload, Document $webhook, Do
                 'X-' . APP_NAME . '-Webhook-Name: ' . $webhook->getAttribute('name', ''),
                 'X-' . APP_NAME . '-Webhook-User-Id: ' . $user->getId(),
                 'X-' . APP_NAME . '-Webhook-Project-Id: ' . $project->getId(),
-                'X-' . APP_NAME . '-Webhook-Signature: ' . $webhook->getAttribute('signature', 'not-yet-implemented'),
+                'X-' . APP_NAME . '-Webhook-Signature: ' . $webhook->getAttribute('signature', $signature),
```

Comment:
```suggestion
                'X-' . APP_NAME . '-Webhook-Signature: ' . $signature,
```

We don't wanna ask the DB here 👍🏻

Reply:
Opsss